tests/e2e/connection.go

```diff
@@ -71,7 +72,10 @@ func connect(ctx context.Context, vs *vSphere) {
 	}
 	framework.Logf("Creating new client session since the existing session is not valid or not authenticated")
 	err = vs.Client.Logout(ctx)
-	framework.Logf("Error from logging out from session is: %v", err)
+	if err != nil {
+		framework.Logf("Ignoring the log out error: %v", err)
+	}
+	framework.Logf("Creating new VC session")
```

Comment:
Can you print a unique message - this message is same as the message in line 64

Reply:
Done